.solhint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "solhint:recommended",
+  "rules": {
+    "indent": [4],
+    "not-rely-on-time": "off",
```

Comment:
What is the reason for not taking the default reccomendations for not-rely-on-time and mark-callable-contracts?

Reply:
Because we have to rely on time. Not using it is not an option. The alternative would be to disable solhint for every line where we access the timestamp, but if we do this everywhere we should just turn it off entirely.

As for marking callable contracts I don't think there is much use here. We should consider every address untrusted and callbable anyway therefore it makes no sense to mark all of them.